src/gui_common/tooltip/microbe_editor/SelectionMenuToolTip.cs

```diff
@@ -99,20 +137,41 @@ public override void _Ready()
         nameLabel = GetNode<Label>(NameLabelPath);
         mpLabel = GetNode<Label>(MpLabelPath);
         descriptionLabel = GetNode<Label>(DescriptionLabelPath);
+        processesDescriptionLabel = GetNode<RichTextLabel>(ProcessesDescriptionLabelPath);
         modifierInfoList = GetNode<VBoxContainer>(ModifierListPath);
-        processList = GetNode<VBoxContainer>(ProcessListPath);
+        processList = GetNode<ProcessList>(ProcessListPath);
+
+        modifierInfoScene = GD.Load<PackedScene>("res://src/gui_common/tooltip/microbe_editor/ModifierInfoLabel.tscn");
+        latoBoldFont = GD.Load<Font>("res://src/gui_common/fonts/Lato-Bold-Smaller.tres");
 
         UpdateName();
         UpdateDescription();
+        UpdateProcessesDescription();
+        UpdateMpCost();
         UpdateLists();
     }
 
+    public override void _Notification(int what)
+    {
+        if (what == NotificationTranslationChanged)
+        {
+            UpdateProcessesDescription();
+        }
+
+        if (what == NotificationResized)
+        {
+            // A workaround to get RichTextLabel's height properly updated on size change
```

Comment:
Is this a known bug in Godot? Should we open an issue in the thrive repo and link to that here about improving this?
I imagine we'll use rich text elsewhere as well and that might also need this workaround...

Reply:
I think it is, I'm using RichTextLabel's [fit_content_height ](https://docs.godotengine.org/en/stable/classes/class_richtextlabel.html#class-richtextlabel-property-fit-content-height) property which in itself is a workaround as noted in the docs and will be removed in future releases (possibly replaced in 4.0) so I don't think we have to open an issue(?). Enabling the property is kind of necessary, otherwise rich text label could hardly be used in the tooltips...

Reply:
Well could you open an issue reminding that this should be revisited once Godot 4 comes out and link that new issue here so that we can remember to do that.

Reply:
Sure opened a new issue #2236